### ISSUE TITLE
Search doesn't find slogans

### ISSUE THREAD

**Issue by aisejohan** [pbelmans/gerby-website#148]
This may not be easily fixable but search terms in slogans don't show up on search results. Example: search for miracle. One doesn't find https://stacks.math.columbia.edu/tag/00R4 Pointed out by Remy.

**Comment by pbelmans** [comment]
Yes, slogans are not part of the HTML which is being searched. That being said

* we could search on slogans and display those results prominently
* we could add the slogan text to the HTML in the fulltext search table

or a combination of these: display slogans with a proper hit on the keywords in just the slogan prominently, and display anything which hits because of a combination of keywords in the text including the slogan the way we do now.


If someone wants to get familiar with Gerby during the workshop, this might be an excellent project to do so!

**Comment by aisejohan** [comment]
Suggest first solution as easier and perhaps more appropriate
